app/src/test/java/com/sdp/movemeet/backend/serialization/MessageSerializerTest.java

```diff
@@ -0,0 +1,24 @@
+package com.sdp.movemeet.backend.serialization;
+
+import com.sdp.movemeet.models.Message;
+
+import org.junit.Test;
+
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+public class MessageSerializerTest {
+
+    @Test
+    public void serializeDeserializeMessageWorks() {
+        Message message = new Message("user", "my message", "user id", "my url");
+        MessageSerializer serializer = new MessageSerializer();
+        Map<String, Object> serialized = serializer.serialize(message);
+        Message deserializedMessage = serializer.deserialize(serialized);
+        assertEquals(message.getMessageText(), deserializedMessage.getMessageText());
+        assertEquals(message.getMessageUser(), deserializedMessage.getMessageUser());
+        assertEquals(message.getMessageUserId(), deserializedMessage.getMessageUserId());
+        assertEquals(message.getImageUrl(), deserializedMessage.getImageUrl());
```

Comment:
Ok nice to see how you can test this serializer! 👍 By the way, as you know the constructor of Message has changed... I will have to be careful to modify this test when I merge my branch.

Reply:
Yes true !